src/jit/gschecks.cpp

```diff
@@ -413,7 +413,9 @@ void Compiler::gsParamsToShadows()
             lvaTable[shadowVar].lvBaseType  = varDsc->lvBaseType;
         }
 #endif
+
         lvaTable[shadowVar].lvRegStruct = varDsc->lvRegStruct;
+        lvaTable[shadowVar].lvExactSize = varDsc->lvExactSize;
```

Comment:
I see that Pat added the line you refer to long after I added mine (when I started working on this), so it's probably not needed now.